### PR TITLE
Update readme.txt

### PR DESCRIPTION
**Pull request by Varorbc**

### CODE REVIEW

**Comment by niemyjski** [comment]
Thanks for the PR, can you please let us know what issue this addressed.

**Comment by Varorbc** [comment]
image

File: src/Platforms/Exceptionless.NLog/readme.txt [hunk 1]
```diff
@@ -24,26 +24,32 @@ the minLevel on the Exceptionless target to Trace so that you can control log le
 client configuration settings. Also, you can call Configuration.SetDefaultMinLogLevel to control the default
 minimum log level that will be used until the client retrieves settings from the server.
 
-<nlog>
+<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      autoReload="true"
```
Comment (niemyjski) [inline src/Platforms/Exceptionless.NLog/readme.txt]: Could you please remove the following:

```
      autoReload="true"
       internalLogLevel="Warn"
       internalLogFile="Logs/internal-nlog.txt"
```

File: src/Platforms/Exceptionless.NLog/readme.txt [hunk 2]
```diff
@@ -24,26 +24,32 @@ the minLevel on the Exceptionless target to Trace so that you can control log le
...
-      <field name="identity" layout="${identity}" />
-      <field name="windows-identity" layout="${windows-identity:userName=True:domain=False}" />
-      <field name="process" layout="${processname}" />
-    </target>
-  </targets>
-  
-  <rules>
-    <logger name="*" minlevel="Trace" writeTo="exceptionless" />
-  </rules>
+    <add assembly="Exceptionless.NLog"/>
```
Comment (niemyjski) [inline src/Platforms/Exceptionless.NLog/readme.txt]: Can you please revert the leading whitespace changes (I believe the tabs on line 35 to 49) and trailing on 43

**Comment by niemyjski** [comment]
Thanks for the PR!

**Pull request merged**